lib/rest/api/v2010/account/incomingPhoneNumber.js

```diff
@@ -106,65 +106,7 @@ IncomingPhoneNumberList = function IncomingPhoneNumberList(version, accountSid)
    * @param {Function} [callback] - Function to process each record
    */
   /* jshint ignore:end */
-  IncomingPhoneNumberListInstance.each = function each(opts, callback) {
-    if (_.isFunction(opts)) {
-      callback = opts;
-      opts = {};
-    }
-    opts = opts || {};
-    if (opts.callback) {
-      callback = opts.callback;
-    }
-    if (_.isUndefined(callback)) {
-      throw new Error('Callback function must be provided');
-    }
-
-    var done = false;
-    var currentPage = 1;
-    var currentResource = 0;
-    var limits = this._version.readLimits({
-      limit: opts.limit,
-      pageSize: opts.pageSize
-    });
-
-    function onComplete(error) {
-      done = true;
-      if (_.isFunction(opts.done)) {
-        opts.done(error);
-      }
-    }
-
-    function fetchNextPage(fn) {
-      var promise = fn();
-      if (_.isUndefined(promise)) {
-        onComplete();
-        return;
-      }
-
-      promise.then(function(page) {
-        _.each(page.instances, function(instance) {
-          if (done || (!_.isUndefined(opts.limit) && currentResource >= opts.limit)) {
-            done = true;
-            return false;
-          }
-
-          currentResource++;
-          callback(instance, onComplete);
-        });
-
-        if (!done) {
-          currentPage++;
-          fetchNextPage(_.bind(page.nextPage, page));
-        } else {
-          onComplete();
-        }
-      });
-
-      promise.catch(onComplete);
-    }
-
-    fetchNextPage(_.bind(this.page, this, _.merge(opts, limits)));
-  };
+  IncomingPhoneNumberListInstance.each = version.each;
```

Comment:
Had to modify the generated file to test out the new pagination function in oai-generator as a temporary solution (waiting the fix in DI-2334)